proxy/InkAPI.cc

```diff
@@ -9643,3 +9644,11 @@ TSHttpTxnPostBufferReaderGet(TSHttpTxn txnp)
   HttpSM *sm = (HttpSM *)txnp;
   return (TSIOBufferReader)sm->get_postbuf_clone_reader();
 }
+
+tsapi TSReturnCode
+TSSslOCSPCallbackSet(TSSslContext ctx)
+{
+  if (SSL_CTX_set_tlsext_status_cb(reinterpret_cast<SSL_CTX *>(ctx), ssl_callback_ocsp_stapling))
```

Comment:
You forget the ATS-mandatory unnecessary braces here.  @zwoop owns stock in Bracemakers Incorporated, his whole retirement plan hinges on us using unnecessary braces.

Reply:
The excess braces are part of the clang formatting requirements. It was actually Bryan who used a variant of that to put in braces for all conditional statements.